iron-validatable-behavior.html

```diff
@@ -59,37 +58,11 @@
         notify: true,
         reflectToAttribute: true,
         type: Boolean,
-        value: false
-      },
-
-      /**
-       * This property is deprecated and should not be used. Use the global
-       * validator meta singleton, `Polymer.IronValidatableBehaviorMeta` instead.
-       */
-      _validatorMeta: {
-        type: Object
+        value: false,
+        observer: '_invalidChanged'
       },
-
-      /**
-       * Namespace for this validator. This property is deprecated and should
-       * not be used. For all intents and purposes, please consider it a
-       * read-only, config-time property.
-       */
-      validatorType: {
```

Comment:
(same thing as [this thread in iron-validator-behavior](https://github.com/PolymerElements/iron-validator-behavior/pull/25/files#r108788061))